applications/asset_tracker/src/main.c

```diff
@@ -620,6 +627,23 @@ static void send_modem_at_cmd_work_fn(struct k_work *work)
 	k_sem_give(&modem_at_cmd_sem);
 }
 
+static void gps_time_set(struct gps_pvt *gps_data)
+{
+	struct tm gps_time = { 0 };
+
+	/* Change datetime.year and datetime.month to accommodate the
+	 * correct input format.
+	 */
+	gps_time.tm_year = gps_data->datetime.year - 1900;
+	gps_time.tm_mon = gps_data->datetime.month - 1;
+	gps_time.tm_mday = gps_data->datetime.day;
+	gps_time.tm_hour = gps_data->datetime.hour;
+	gps_time.tm_min = gps_data->datetime.minute;
+	gps_time.tm_sec = gps_data->datetime.seconds;
```

Comment:
A bit more efficient way to achieve the same (not a request for change in this PR):
```
/* Change datetime.year and datetime.month to accommodate the
 * correct input format.
 */
struct tm gps_time = {
	.tm_year = gps_data->datetime.year - 1900,
	.tm_mon = gps_data->datetime.month - 1,
	.tm_mday = gps_data->datetime.day,
	.tm_hour = gps_data->datetime.hour,
	.tm_min = gps_data->datetime.minute,
	.tm_sec = gps_data->datetime.seconds,
};
```